cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go

```diff
@@ -549,14 +614,29 @@ type mockConfiguration struct {
 	descriptorFilenames []string
 	fileURIs            []string
 	images              []domain.Image
+	disks               []domain.Disk
 }
 
 func setupMocksAndRun(mockCtrl *gomock.Controller, params *ovfdomain.OVFImportParams, wfPath string, descriptor *ovf.Envelope, mockConfig mockConfiguration) (daisyutils.DaisyWorker, error) {
+	params.InstanceNames = strings.ToLower(strings.TrimSpace(params.InstanceNames))
+	params.MachineImageName = strings.ToLower(strings.TrimSpace(params.MachineImageName))
 	expectedParams := *params
 	expectedParams.OsID = mockConfig.expectedOS
 	expectedParams.Deadline = params.Deadline.Add(-1 * instanceConstructionTime)
 	mockStorageClient := mocks.NewMockStorageClientInterface(mockCtrl)
 	mockComputeClient := mocks.NewMockClient(mockCtrl)
+	mockStorageObject := mocks.NewMockStorageObject(mockCtrl)
+
+	mockComputeClient.EXPECT().CreateDisk(*params.Project, params.Zone, gomock.Any()).Return(nil).AnyTimes()
+
+	var imageName string
+	if params.InstanceNames == "" {
+		imageName = params.MachineImageName
+	} else {
+		imageName = params.InstanceNames
+	}
+	imageName += "-boot-image"
```

Comment:
I think `if` is not required here, you will always have `InstanceNames` specified due to the definition part.